docs/getting-started/prefect-explained.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Prefect Explained
+description: Learn core Prefect concepts.
+tags:
+    - getting started
+    - explanation
+    - overview
+search:
+  boost: 2
+---
+
+# Prefect Explained
+
+Prefect is a Pythonic framework for orchestrating and observing workflows.
+
+You install the [`prefect` package](/getting-started/installation/) to gain access to the Prefect client that contains the Python SDK and CLI commands.
+
+The Prefect client talks to a Prefect server - either the managed [Prefect Cloud platform](/cloud/) or a [self-hosted server instance](/guides/host/) backed by SQLite or PostgreSQL.
+
+Prefect Cloud provides [free and paid plans](https://www.prefect.io/pricing) with additional features useful for teams and production workloads.
+
+Both a self-hosted server and Prefect Cloud provide a UI for visualizing and managing your workflows.
+
+![Cloud Dashboard](/img/ui/cloud-dashboard.png)
+
+Here are the three major concepts you need to know to build a production workflow with Prefect:
+
+- Flows
+- Deployments
+- Work pools
+
+## Flows
+
+A Prefect [flow](/concepts/flows/) is a Python function that defines a workflow and is decorated by `@flow`.
+When this function runs, Prefect can track the state of the workflow and visualize it in the UI.
+
+As shown below, when a flow is referenced as the entrypoint in a deployment it can be scheduled.
+
+![Dependency graph in the UI](/img/ui/dependency-graph.png)
+
+A flow is defined in Python code, stored in the location specified in a deployment, and represented on the server.
+
+## Work pools
+
+A [work pool](/concepts/work-pools/) provides presets for infrastructure.
+The work pool type could be a local subprocess, a Docker container, a Kubernetes cluster, or serverless cloud infrastructure such as AWS ECS, Google Cloud Run, VertexAI, or Azure Container Instances.
+Prefect Cloud provides two special work pool categories: Prefect managed work pools and push work pools on serverless cloud infrastructure (ECS, Google Cloud Run, or Azure Container Instances).
+
+Work pools are created in the UI or CLI and live on the server.
+
+## Deployments
+
+Creating a [deployment](/concepts/deployments/) makes your flow schedulable.
+The deployment contains metadata needed to run the flow in production.
+
+A [deployment specifies](/guides/prefect-deploy/) a single entrypoint flow function, a deployment name, a work pool, and the location of the flow code to be pulled at runtime.
+
+Flow code storage options include a Docker registry with the flow code baked into an image, git-based storage such as a GitHub repository, or cloud-provider storage such as AWS S3.
+
+Deployments are created in Python code with `deploy`, via the CLI interactively with `prefect deploy`, or applied from a `prefect.yaml` file. Deployments live on the server.
```

Comment:
```suggestion
Deployments are created in Python code with `flow.deploy()`, via the CLI interactively with `prefect deploy`, or applied from a `prefect.yaml` file. Deployments live on the server.
```

Reply:
Or with the `deploy` function - less common, but leaving as `deploy` captures both options, I think.